server/portal/templates/includes/cms_menu.html

```diff
@@ -1,17 +1,20 @@
-{% load i18n cache %}
+{% load i18n menu_tags cache %}
+{# FAQ: This would be called by `nav_cms.html`, but instead that file loads the CMS markup via AJAX #}
```

Comment:
__Explanation__: This file is __not__ used. It is updated merely for consistency with CMS. _It __might__ be deleted in the future, if it is confirmed to be unlikely to ever serve a purpose._